package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "esn-api-client",
+  "version": "0.0.1",
+  "description": "JS library for OpenPaaS ESN APIs",
+  "main": "src/index.js",
```

Comment:
If you use Webpack to bundle & build the library, this field should point to the built path.